README.md

```diff
@@ -1,13 +1,18 @@
+Yoshino platform
+==============
```

Comment:
You might as well use three `===` signs here (and three `---` dashes below), _or_ keep them exactly aligned :wink:

Reply:
Perfectionist inside me has to get it alligned. Otherwise my brain will go nuts :)

So i will fix it :)